tests/components/mobile_app/test_webhook.py

```diff
@@ -24,10 +24,10 @@
 from .const import CALL_SERVICE, FIRE_EVENT, REGISTER_CLEARTEXT, RENDER_TEMPLATE, UPDATE
 
 from tests.common import async_capture_events, async_mock_service
-from tests.components.conversation.conftest import mock_agent
+import tests.components.conversation.conftest
```

Comment:
We shouldn't import fixtures. Preferably we refactor this first in a separate PR so we don't forget about it.

The fixture should be made a function that can be imported in fixtures in other modules and used. Example:

https://github.com/home-assistant/core/blob/045dc3f1fb37a4eb462f1caa5b7e84ab150b9703/tests/components/conftest.py#L59-L64

https://github.com/home-assistant/core/blob/045dc3f1fb37a4eb462f1caa5b7e84ab150b9703/tests/components/tts/common.py#L44-L49